modules/python/python-main.c

```diff
@@ -72,7 +72,7 @@ _py_construct_main_module(void)
   module_dict = PyModule_GetDict(module);
   if (PyDict_GetItemString(module_dict, "__builtins__") == NULL)
     {
-      PyObject *builtins_module = PyImport_ImportModule("__builtin__");
```

Comment:
Could you include `compat-python.h` explicitly (or `python-module.h`)?

`compat-python.h` is included through a non-trivial chain:


`python-main.c` <-- `python-helpers.h` <-- `python-module.h` <-- `compat-python.h`